boards/nrf51dk/src/main.rs

```diff
@@ -242,6 +244,18 @@ pub unsafe fn reset_handler() {
         96/8);
     nrf51::trng::TRNG.set_client(rng);
 
+    let aes = static_init!(
+        capsules::symmetric_encryption::Crypto<'static, nrf51::aes::AesECB>,
+        capsules::symmetric_encryption::Crypto::new(&mut nrf51::aes::AESECB,
+                                                    kernel::Container::create(),
+                                                    &mut capsules::symmetric_encryption::BUF,
+                                                    &mut capsules::symmetric_encryption::BUF,
```

Comment:
I will add another buffer then, because these are needed initially for the TakeCell i.e. such as it is not "None"